### PR TITLE
rsa_psa_sha256 should have salt length of 32

### PR DESCRIPTION
**Pull request by nyaxt**
Not sure what we can do about the test. Iterate through the interface{} and make []uint8 all string?

### CODE REVIEW

File: go/signedexchange/signedexchange_test.go [hunk 1]
```diff
@@ -158,12 +156,10 @@ func TestSignedExchange(t *testing.T) {
...
-		t.Errorf("len(got) vs len(want): got: %v, want: %v", len(got), len(want))
-	}
-	if got != want {
-		t.Errorf("WriteExchangeFile:\ngot %v\nwant %v", got, want)
+	var decoded interface{}
+	handle := &codec.CborHandle{}
+	if err := codec.NewDecoder(&buf, handle).Decode(&decoded); err != nil {
+		t.Fatal(err)
 	}
+	t.Errorf("%#+v", decoded) // FIXME: dousiyou
```
Comment (hajimehoshi) [inline go/signedexchange/signedexchange_test.go:164]: dousiyou ne

**Review by hajimehoshi — CHANGES_REQUESTED** [review]

File: go/signedexchange/signedexchange_test.go (hunk 1 above)
Comment (hajimehoshi) [reply go/signedexchange/signedexchange_test.go:164]: https://github.com/WICG/webpackage/pull/119/commits/38c496415ca945dd369fcf3576f2a5416d63f9fb ko shita

Comment (nyaxt) [reply go/signedexchange/signedexchange_test.go:164]: lgtm myside

**Comment by hajimehoshi** [comment]
Adopted by https://github.com/hajimehoshi/webpackage/commit/99b672c46ca1899e630688701fedf8a4b3c92f7f

**Pull request closed without merging**